### PR TITLE
non-resonant HH Run3

### PR DESCRIPTION
**Pull request by acarvalh**
These are the full set of planned samples for HH for Run3

### CODE REVIEW

File: bin/Powheg/production/Run3/13p6TeV/Higgs/gg_HH_HEFT/powheg_ggHH_BM10_JHEP04.input [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+! OPTIONAL PARAMETERS
+
+! scale settings:
+#fixedscale   1      ! (dafault 0) fixed scale uses muren = mufac = 2*m_H
+facscfact   1d0      ! (default 1d0) fac scale factor: mufact = muref * facscfact, muref=mHH/2
+renscfact   1d0      ! (default 1d0) ren scale factor: muren  = muref * renscfact, muref=mHH/2
+
+! Born, plots and powheg strategy:
+bornonly          1  ! (default 0) if 1 do Born only
+LOevents          1  !
```
Comment (menglu21) [inline bin/Powheg/production/Run3/13p6TeV/Higgs/gg_HH_HEFT/powheg_ggHH_BM10_JHEP04.input:70]: is it possible to run at NLO order

**Comment by menglu21** [comment]
do i understand correctly that the only difference among those cards is the five couplings. the comment i left is for all the cards, whether if it's possible to run at NLO

**Comment by acarvalh** [comment]
Hi @menglu21 , we are really sorry to be bugging you again here

These samples are needed by TSG studies,  trigger reviews are approaching (week of February 13th), and production can only start after this PR merged

The last commits since 12 Dec were mostly cosmetics and warnings to bookkeep

Do you think everything is ok here?
thanks!!!!

**Comment by Saptaparna** [comment]
Hi @menglu21, did you have any other comments here?

**Comment by mlizzo** [comment]
Cards look good to me, I'll merge this PR

**Pull request merged**